README.md

```diff
@@ -360,17 +358,11 @@ Next, create the role with the permissions JSON file we just created, along with
 
 
 ```
-aws iam create-role --role-name IoTCloudWatchRole --assume-role-policy-document file://./templates/cwl_role_trust.json
+export CWL_ROLE_ARN=$(aws iam create-role --role-name IoTCloudWatchRole --assume-role-policy-document file://./templates/cwl_role_trust.json --query 'Role.Arn')
```

Comment:
I haven't tested this, but from memory doing the query without setting `--output text` will include the quotation marks in the result, which means it won't work with future function calls. Have you tested that this works correctly?

Reply:
fixed